modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/SystemDataSets.tsx

```diff
@@ -21,6 +27,107 @@ import openDefaultFailureToast from './utils/openDefaultFailureToast';
 import openDefaultSuccessToast from './utils/openDefaultSuccessToast';
 import {IDataSet, ISystemDataSet} from './utils/types';
 
+interface IFrontendDataSetContext {
+	onSelect: Function;
+	selectItems: Function;
+	selectable: boolean;
+	selectedItemsKey: keyof ISystemDataSet;
+	selectedItemsValue: Array<any>;
+}
+
+const SystemDataSetModalList = ({
+	frontendDataSetContext,
+	items,
+}: {
+	frontendDataSetContext: any;
+	items: Array<ISystemDataSet>;
+}) => {
+	const {
+		onSelect,
+		selectItems,
+		selectable,
+		selectedItemsKey,
+		selectedItemsValue,
+	} = useContext(frontendDataSetContext) as IFrontendDataSetContext;
+
+	return (
+		<ClayList>
+			{items.map((item) => {
+				return (
+					<ClayList.Item
+						className={classNames({
+							disabled: item.customized,
+							selectable,
+							selected: selectedItemsValue?.includes(
+								item[selectedItemsKey]
+							),
+						})}
+						flex
+						key={item.name}
+						onClick={() => {
+							if (selectable) {
+								selectItems(item[selectedItemsKey]);
+
+								onSelect({selectedItems: [item]});
+							}
+						}}
+					>
+						<ClayList.ItemField className="justify-content-center selection-control">
+							<ClayRadio
+								checked={
+									selectedItemsValue
+										? selectedItemsValue
+												.map((element) =>
+													String(element)
+												)
+												.includes(
+													String(
+														item[selectedItemsKey]
+													)
+												)
+										: false
+								}
+								onChange={() => {}}
+								value=""
+							/>
+						</ClayList.ItemField>
+
+						<ClayList.ItemField>
+							<ClaySticker displayType="dark">
+								<ClayIcon symbol={item.symbol} />
+							</ClaySticker>
+						</ClayList.ItemField>
+
+						<ClayList.ItemField
+							className="justify-content-center"
+							expand
+						>
+							<ClayList.ItemTitle>
+								{item.title}
+							</ClayList.ItemTitle>
+
+							<ClayList.ItemText>
+								{item.description}
+							</ClayList.ItemText>
+						</ClayList.ItemField>
+
+						{item.customized && (
+							<ClayList.ItemField>
+								<ClayLabel
+									className="used-label"
+									displayType="warning"
+								>
+									{Liferay.Language.get('used')}
```

Comment:
I doubt about this wording. @herrardo what do you think?

The notion of "Used" system data set might be confusing because usage has different meanings. Ours is that system data set is already customized in the DSM. Other meaning is that the system data set is being used in a page (probably true for all of them).

Not sure if we made the naming exercise before. I can think of terms like "Customized" or even "In use" (at least this wording already exists)

Reply:
I think you are right and that could be confusing. As long as the button to perform the actions is "Create", I would use "Created".

Reply:
I like that naming, it's less complicated than "customized"

@thektan please change the label, the css and I'd say the json key to reflect this naming.